### PR TITLE
Add placeholders for weight and reformate code

### PR DESCRIPTION
**Pull request by ExusiaiMoe**
Add placeholder `%extrahardmode_weight%` and `%extrahardmode_max_weight%`
Also did a reformate of the whole project using the code style in assests

### CODE REVIEW

File: src/main/resources/plugin.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ author: RoboMWM
 authors: [Diemex, Big_Scary]
 #ensure that we are loaded after multiworld plugins
 softdepend: [Multiverse-Core, My_Worlds, MystCraft, Transporter, WorldGuard, Prism, LogBlock, CoreProtect, HawkEye, Crackshot]
+depend: [PlaceholderAPI]
```
Comment (RoboMWM) [inline src/main/resources/plugin.yml:10]: this is a yikes, cuz ppl like me don't use this plugin and thus will cause EHM to not load on such servers.

**Comment by RoboMWM** [comment]
Save code refactor for another PR.

I haven't gone over the styling in this yet, if you'd like to suggest changes to such a file (if it exists? Idk if it does I never checked for one, and I'm currently only familiar with intellij's .editorconfig - albeit I'm open to any other ways to communicate convention)

**Comment by ExusiaiMoe** [comment]
I just used this file 
https://github.com/MLG-Fortress/ExtraHardMode/blob/34b704e56f77d34157966c56c33b21c9e6fe1a70/src/main/assets/IntelliJ%20-%20import%20for%20our%20codestyle.jar
And ok, I'll make placeholder a option feature

**Comment by ExusiaiMoe** [comment]
Done. Also I advice for a clean up of those unused competitive classes

**Review by RoboMWM — APPROVED** [review]

**Comment by RoboMWM** [comment]
Ah, I guess the previous maintainer exported his intellij settings? Interesting.

Unless there are other recommended formats I'll get an .editorconfig in here. Ideally I want to be consistent with the existing code so it might need some adapting from the other project I currently also publish one in (GriefPrevention).

**Comment by ExusiaiMoe** [comment]
And maybe open the discussions section for more convenient conmunications?

**Comment by RoboMWM** [comment]
yea there's an IRC channel (preferred) or dumcord. Idk if I have it in the readme.

An issue for discussing should be fine given the general activity of this plugin but I could enable that section too.

**Comment by ExusiaiMoe** [comment]
Seems none is on readme, lol

**Comment by RoboMWM** [comment]
if you want to add it'd basically be https://robomwm.com/chat more or less.

**Pull request merged**